src/test/resources/blackbox/aztec-1/Historico.txt

```diff
@@ -0,0 +1 @@
+Histórico
```

Comment:
🇧🇷 🇵🇹

Reply:
one of the strings which failed in the beginning ^^

Reply:
I must admit these accents in our language doesn't make our lives easier hehe.